docs/admin/logging/etwlogs.md

```diff
@@ -0,0 +1,69 @@
+<!--[metadata]>
++++
+title = "ETW logging driver"
+description = "Describes how to use the etwlogs logging driver."
+keywords = ["ETW, docker, logging, driver"]
+[menu.main]
+parent = "smn_logging" 
+weight=2
++++
+<![end-metadata]-->
+
+
+# ETW logging driver
+
+The ETW logging driver forwards container logs as ETW events. 
+ETW stands for Event Tracing in Windows, and is the common framework
+for tracing applications in Windows. Each ETW event contains a message
+with both the log and its context information. A client can then create
+an ETW listener to listen to these events. 
+
+The ETW provider that this logging driver registers with Windows, has the 
+GUID identifier of: '{a3693192-9ed6-46d2-a981-f8226c8363bd}'. A client creates an 
```

Comment:
I think we should put the GUID here in back-ticks to make it stand out more;

`{a3693192-9ed6-46d2-a981-f8226c8363bd}`